mismi-s3/test/Test/IO/Mismi/S3/Commands.hs

```diff
@@ -151,6 +154,35 @@ prop_upload d l = withLocalAWS $ \p a -> do
   r <- read a
   pure $ r === Just d
 
+prop_uploadFromURL :: Property
```

Comment:
Need to add a test which checks the actual value of the s3 object after streaming (we have had issues in the past with streaming / conduit corrupting data)